pkg/reconciler/runner.go

```diff
@@ -33,6 +33,7 @@ type Runner struct {
 	Logger      logr.Logger
 }
 
+// TODO: make sure that model server container (deployment) is ready before creating kepler daemonset
```

Comment:
why is this needed? Shouldn't kepler retry for some time?

Reply:
Current code kepler does not retry if not found model server. All is done at configuration phase.

Reply:
In a different PR, we can write a reconciler that waits for a resource to be available. If modelserver is enabled, inject that before kepler daemonset is created.